proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -172,6 +172,25 @@ message GetWorkflowExecutionRawHistoryV2Response {
   repeated int64 history_node_ids = 4;
 }
 
+message GetWorkflowExecutionRawHistoryRequest {
+  reserved 1;
```

Comment:
this is a new type. do you still need `reserved 1` keyword?

Reply:
Good catch.